packages/blade/src/components/Modal/Modal.web.tsx

```diff
@@ -106,13 +106,22 @@ const Modal = ({
   size = 'small',
   accessibilityLabel,
 }: ModalProps): React.ReactElement => {
-  const { theme } = useTheme();
+  const { theme, platform } = useTheme();
   const [footerHeight, setFooterHeight] = useState(0);
   const { isMounted, isVisible } = usePresence(isOpen, {
     transitionDuration: theme.motion.duration.xmoderate,
     initialEnter: true,
   });
 
+  // Warn consumer if modal is opened on mobile
```

Comment:
Minor: probably an overkill 😅

Reply:
Should be okay, right? Kamlesh had also asked to add this. In fact he suggested we throw an error but we took the call of throwing a warning instead.

Reply:
Though this will also throw warnings on tablet size

Reply:
Should be fine. We want every non-desktop platform to use bottomsheet only